src/Commands/BranchDeployForgeCommand.php

```diff
@@ -176,10 +176,13 @@ protected function createDatabase(Server $server, Site $site): void
 
     protected function updateEnvFile(Server $server, Site $site): void
     {
+        $branch = $this->getBranch();
+        $appEnv = in_array(strtolower($branch), ['main', 'production']) ? $branch : 'th-' . $branch;
```

Comment:
why would you have this condition for main, you don't want it

Reply:
we also have this weird condition `testingEnv` is it something we use? maybe we should standardized the options for this property

Reply:
I would not use the branch name then, either the same format as for the rest either `getFullEnvName`
or

Reply:
I changed